modules/stats/trade.py

```diff
@@ -92,22 +120,30 @@ def configure_stoploss(self, ohlcv: dict, data_dict: dict) -> None:
             self.sl_sell_time, self.sl_ratio = self.trailing_stoploss(data_dict, ohlcv['time'])
 
     def update_max_drawdown(self) -> None:
-        if self.capital < self.lowest_seen_price:
-            self.lowest_seen_price = self.capital
-            self.max_seen_drawdown = self.profit_ratio
-
-        is_new_drawdown_period = self.capital > self.seen_peak_capital
-        if is_new_drawdown_period:
-            self.seen_peak_capital = self.capital
-            self.temp_seen_drawdown_price = self.capital
-        elif self.capital < self.temp_seen_drawdown_price:
-            self.temp_seen_drawdown_price = self.capital
-            self.set_drawdown_if_lower()
-
-    def set_drawdown_if_lower(self):
-        if self.temp_seen_drawdown_price < self.max_seen_drawdown_price:
-            self.max_seen_drawdown_price = self.temp_seen_drawdown_price
-            self.max_seen_drawdown = self.temp_seen_drawdown_price / self.seen_peak_capital
+        """
+        Updates max seen drawdown.
+
+        :return: None
+        :rtype: None
+        """
+        curr_open_capital = self.open_value * self.currency_amount
+        curr_lowest_capital = self.low_value * self.currency_amount
+        
+        # Check for new drawdown period
+        if curr_open_capital > self.temp_highest_seen_capital:
```

Comment:
```suggestion
        if curr_open_capital > self.temp_highest_seen_capital:
```
        if curr_total_capital > self.temp_highest_seen_capital:
Open suggests only open positions, total capital/balance/total would be more appropriate